tests/test_invalid_target.py

```diff
@@ -0,0 +1,20 @@
+import os
+import subprocess
+
+from utils.command import build_command
```

Comment:
Looks good @abrugaro . 
We can rename the file to more generic like test_invalid_fields.py and add more test cases for wrong inputs or missing inputs and do validations .

For example , missing --input

Reply:
Solved in 136e8508c22e5d201315e774978fd52d1f678ba7.